src/Schema.luna

```diff
@@ -12,6 +12,8 @@ class Schema:
         wrapper' = wrapper
         SchemaVal wrapper'
 
+    def shortRep: "Schema"
```

Comment:
Not sure how short `shortRep` should be, but we might want to add additional type information here as well. `Schema` is basically a type of a table (i.e. sequence of fields).